gitops/plnsvc/base/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - https://raw.githubusercontent.com/roming22/pipelines-service/my/gitops/gitops/plnsvc/base/namespace.yaml
+  - https://raw.githubusercontent.com/roming22/pipelines-service/my/gitops/gitops/plnsvc/base/kcp-manager.yaml
+
```

Comment:
same as previously:
- Why is it pointing to your own repo?
- Why are the files not directly in the directory?

Reply:
Fixed